framework/source/class/qx/event/message/Bus.js

```diff
@@ -8,8 +8,7 @@
      2007 Christian Boulanger
 
    License:
-     LGPL: http://www.gnu.org/licenses/lgpl.html
-     EPL: http://www.eclipse.org/org/documents/epl-v10.php
+     MIT: https://opensource.org/licenses/MIT
      See the LICENSE file in the project's top-level directory for details.
```

Comment:
I will fix this with the next path which changes all the 1&1 files ... it seems that calling the file LICENSE is 'the right thing'